### ISSUE TITLE
Quiz about best way to define SLAs is potentially ambiguous

### ISSUE THREAD

**Issue by MattDodsonEnglish** [grafana/k6-learn#10]
"Setting load profiles" has a quiz question that asks about "The best way to define baseline metrics". Is this question too vague? Won't different SLAs have different key metrics? For example, if I wanted define an SLA on latency, wouldn't it also make sense to use ramping-vus with thresholds?

 But even for one metric, how confidently can we say that one executor is "best" for it?

> We're looking to establish some baseline metrics to define an SLA for an existing service, what is the quickest way to achieve this?
>  A : Use the `shared-iterations` executor to run through 1,000 requests.
>
> B: Use the `constant-arrival-rate` to see how many virtual users it takes to maintain a constant 50 requests per second (RPS).
> 
> C: Use the `externally-controlled` executor to start k6 in server mode to have your _sweet_ Bash script ramp up virtual users.

**Comment by javaducky** [comment]
This may be splitting hairs, but the question is related to *quickest* way to establish some baseline. I _believe_ my thinking may have been that this is the simplest (and default IIRC) executor and therefore the easiest to create a script to get _some_ kind of test going. I definitely didn't intend to imply this was the *best* executor for establishing SLAs for any particular metric.

Call it a "trick question" ;)

**Comment by MattDodsonEnglish** [comment]
Hmm, true, but either way we should be careful to use absolutes: Perhaps the quickest way is to ask your neighbor, or query your server logs. And if we keep it to k6, can we be sure that this is the quickest executor? For what SLA metric?

I propose a making a new question that's grounded in a specific problem. "SLA" is too big and general a concept.

**Comment by dgzlopes** [comment]
Just passing by 👀 

Given that the ones reading this will probably be users/devs who want to learn more about it, it may make sense to talk about SLOs. SLAs are very business related, and the numbers that you put are usually made up (in the sense that it's an non-real number that you don't expect to break)

**Comment by MattDodsonEnglish** [comment]
Yes that's a good point. Anyway, shouldn't an SLA have a higher error budget than the true SLO? So I guess the tester is more interested in finding the SLO to set a baseline.

Since we're talking about Service level *s, in the context of k6 tests, is there any appreciable difference between a metric and an SLI?

**Comment by dgzlopes** [comment]
> Anyway, shouldn't an SLA have a higher error budget than the true SLO? So I guess the tester is more interested in finding the SLO to set a baseline.

Yup :+1: 

> Since we're talking about Service level *s, in the context of k6 tests, is there any appreciable difference between a metric and an SLI?

The metric is the indicator in our case, yeah :thinking:
